src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -183,6 +183,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('key_frame_interval_duration', help='ISO 8601 timespan duration of the key frame interval duration.')
         c.argument('access_token', help='The access token.')
         c.argument('description', help='The live event description.')
+        c.argument('ips', nargs='+', help='The IP allow list.')
```

Comment:
We should let the user know this parameter expects a space-separated list of ips (see other space-separated arguments)